md_core/add_rtt/r_column_bots/add_r_column.py

```
@@ -0,0 +1,54 @@
+#!/usr/bin/python3
+"""
+
+from add_rtt.r_column_bots.add_r_column import add_header_R, header_has_R
+
+
+"""
+# ---
+import wikitextparser as wtp
+from wikitextparser._cell import Cell
```

Ruff (0.8.2)

`wikitextparser._cell.Cell` imported but unused

Remove unused import: `wikitextparser._cell.Cell`

(F401)